src/app/components/media/MediaTypeDisplayName.js

```diff
@@ -71,39 +72,37 @@ export { getMediaTypeDisplayName };
 
 export default function MediaTypeDisplayName({ mediaType }) {
   switch (mediaType) {
-  case 'Claim':
+  case CheckMediaTypes.CLAIM:
     return <FormattedMessage defaultMessage="Text" description="Label to show that the type of media is text" id="media.typeClaim" />;
-  case 'Link':
+  case CheckMediaTypes.LINK:
     return <FormattedMessage defaultMessage="Link" description="Label to show that the type of media is a link" id="media.typeLink" />;
-  case 'UploadedImage':
+  case CheckMediaTypes.UPLOADEDIMAGE:
     return <FormattedMessage defaultMessage="Image" description="Label to show that the type of media is an image" id="media.typeImage" />;
-  case 'UploadedVideo':
+  case CheckMediaTypes.UPLOADEDVIDEO:
     return <FormattedMessage defaultMessage="Video" description="Label to show that the type of media is a video" id="media.typeVideo" />;
-  case 'UploadedAudio':
+  case CheckMediaTypes.UPLOADEDAUDIO:
     return <FormattedMessage defaultMessage="Audio" description="Label to show that the type of media is an audio file" id="media.typeAudio" />;
-  case 'Blank':
+  case CheckMediaTypes.BLANK:
     return <FormattedMessage defaultMessage="Imported fact-check" description="Label to show that the type of item is an imported fact-check" id="media.typeBlank" />;
-  case 'Facebook':
+  case CheckMediaTypes.FACEBOOK:
     return <FormattedMessage defaultMessage="Facebook Post" description="Label to show that the type of media is a Facebook post" id="media.typeFacebook" />;
-  case 'Instagram':
+  case CheckMediaTypes.INSTAGRAM:
     return <FormattedMessage defaultMessage="Instagram Post" description="Label to show that the type of media is an Instagram post" id="media.typeInstagram" />;
-  case 'Telegram':
+  case CheckMediaTypes.TELEGRAM:
     return <FormattedMessage defaultMessage="Telegram" description="Label to show that the type of media is a Telegram message" id="media.typeTelegram" />;
-  case 'Tiktok':
+  case CheckMediaTypes.TIKTOK:
     return <FormattedMessage defaultMessage="TikTok Post" description="Label to show that the type of media is a Tiktok video" id="media.typeTiktok" />;
-  case 'Twitter':
+  case CheckMediaTypes.TWITTER:
     return <FormattedMessage defaultMessage="X (Twitter) Post" description="Label to show that the type of media is a Twitter tweet" id="media.typeTwitter" />;
-  case 'Youtube':
+  case CheckMediaTypes.YOUTUBE:
     return <FormattedMessage defaultMessage="Youtube Video" description="Label to show that the type of media is a Youtube video" id="media.typeYoutube" />;
   case '-':
   default:
     return <React.Fragment>-</React.Fragment>;
   }
 }
 
-MediaTypeDisplayName.MediaTypeShape = PropTypes.oneOf([
-  'Claim', 'Link', 'UploadedImage', 'UploadedVideo', 'UploadedAudio', 'Blank', 'Facebook', 'Instagram', 'Telegram', 'Tiktok', 'Twitter', 'Youtube', '-',
-]);
+MediaTypeDisplayName.MediaTypeShape = PropTypes.oneOf(Object.values(CheckMediaTypes));
```

Comment:
Is this still necessary? Also `MediaTypeShape` sounds a bit misleading, as this is not an object shape `(PropTypes.shape)`. Perhaps we could also stick with `mediaType: CheckPropTypes.mediaType.isRequired` in this file as well for line 108?